src/keyBindings.py

```diff
@@ -0,0 +1,22 @@
+import os
+import ConfigParser
+
+from stateFiles import FPP_DIR
+
+KEY_BINDINGS_FILE = '.fpp.keys'
+
+
+class KeyBindings(object):
+
+    bindings = []
+
+    def __init__(self):
+        """Returns configured key bindings, in the format [[key, command], ...]
+        """
+        configFilePath = os.path.expanduser(
+            os.path.join(FPP_DIR, KEY_BINDINGS_FILE))
+        parser = ConfigParser.ConfigParser()
+        parser.read(configFilePath)
```

Comment:
Nothing, the code is tolerant :-) But it's good practice to add some tests; I will add a test suite.